src/main/java/ua/danit/final_project/configuration/StaticCollection.java

```diff
@@ -0,0 +1,19 @@
+package ua.danit.final_project.configuration;
+
+import ua.danit.final_project.entities.User;
+
+public class StaticCollection {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=84&issues=AWVlsIsL4Bfu85TPT8vo&open=AWVlsIsL4Bfu85TPT8vo)

[ISSUE_KEY]: AWVlsIsL4Bfu85TPT8vo